app/views/brexit_landing_page/show.html.erb

```diff
@@ -14,44 +14,21 @@
 %>
 
 <div class="landing-page__buckets full-page-width-wrapper">
-  <% presented_taxon.buckets.each do |bucket| %>
-    <%= render partial: 'buckets', locals: { bucket: bucket } %>
+  <% if page_variant.variant?("B") %>
+    <%= render partial: 'buckets_b', locals: { buckets: presented_taxon.buckets_b } %>
+  <% else %>
+    <% presented_taxon.buckets_a.each do |bucket| %>
+      <%= render partial: 'buckets_a', locals: { bucket: bucket } %>
+    <% end %>
   <% end %>
 </div>
 
 <div class="full-page-width-wrapper full-page-width-wrapper--landing-page">
-  <div class="govuk-grid-row">
-    <div class="govuk-body govuk-grid-column-two-thirds">
-      <%= render "govuk_publishing_components/components/heading", {
-        text: t('brexit_landing_page.all_brexit_information'),
-        padding: true,
-      } %>
-      <%= render "govuk_publishing_components/components/heading", {
-        text: t('brexit_landing_page.finder_heading'),
-        heading_level: 3,
-        font_size: 19,
-        padding: true
-      } %>
-      <%= render "components/topic-list", {
-        items: presented_taxon.supergroup_sections
-      } %>
-      </div>
-    </div>
-
-  <div class="landing-page__email-wrapper">
-    <svg  class="app-c-email-link__icon" xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 459.334 459.334">
-      <path fill="black" d="M177.216 404.514c-.001.12-.009.239-.009.359 0 30.078 24.383 54.461 54.461 54.461s54.461-24.383 54.461-54.461c0-.12-.008-.239-.009-.359H175.216zM403.549 336.438l-49.015-72.002v-89.83c0-60.581-43.144-111.079-100.381-122.459V24.485C254.152 10.963 243.19 0 229.667 0s-24.485 10.963-24.485 24.485v27.663c-57.237 11.381-100.381 61.879-100.381 122.459v89.83l-49.015 72.002a24.76 24.76 0 0 0 20.468 38.693H383.08a24.761 24.761 0 0 0 20.469-38.694z"/>
-    </svg>
-    <h3 class="govuk-heading-s landing-page__email-title"><%= t('brexit_landing_page.email_intro') %></h3>
-    <%= link_to(t('brexit_landing_page.email'), "/email-signup/?topic=#{presented_taxon.base_path}", {
-      class: "govuk-body govuk-link",
-      data: {
-        module: "track-click",
-        "track-category": "emailAlertLinkClicked",
-        "track-action": presented_taxon.base_path
-      }
-    }) %>
-  </div>
+  <% if page_variant.variant?("B") %>
+    <%= render partial: 'brexit_finders_b', locals: {supergroups: presented_taxon.supergroup_sections, email_path: presented_taxon.base_path } %>
+  <% else %>
+    <%= render partial: 'brexit_finders_a', locals: {supergroups: presented_taxon.supergroup_sections, email_path: presented_taxon.base_path } %>
+  <% end %>
```

Comment:
You can probably do this a bit more concise with: 

`<%= render partial: page_variant.variant?("B") ? 'brexit_finders_b' : 'brexit_finders_a', locals: {supergroups: presented_taxon.supergroup_sections, email_path: presented_taxon.base_path } %>`